src/pybind/mgr/dashboard/services/orchestrator.py

```diff
@@ -62,6 +62,10 @@ def get(self, hostname: str) -> Optional[HostSpec]:
         hosts = [host for host in self.list() if host.hostname == hostname]
         return hosts[0] if hosts else None
 
+    @wait_api_result
+    def get_gather_facts(self, host: Optional[str] = None) -> List[Dict[str, Any]]:
```

Comment:
How about `gather_facts` instead of `get_gather_facts` because I think get and gather have the same meaning here. This call is all about getting/gathering the host facts from orch.

Reply:
Also `hostname` instead of just `host`
```suggestion
    def gather_facts(self, hostname: Optional[str] = None) -> List[Dict[str, Any]]:

Reply:
Done.